src/components/icon-button/IconButton.constants.ts

```diff
@@ -0,0 +1,64 @@
+import { TIconSize } from '../icon';
+import { TTextSize } from '../text';
+
+import { TIconButtonSize } from './IconButton.types';
+
+const DEFAULT_ICON_BUTTON_SIZE: TIconButtonSize = 'standard';
+
+const ICON_BUTTON_SIZES: TIconButtonSize[] = [
+    'xsmall',
+    'small-compact',
```

Comment:
is this "legacy" usage? If not I would suggest using just `compact` 

@nevyangelova and @deanwhillier WDYT?

Reply:
It's following the naming conventions defined in the Design System. We should stick to using the same conventions to avoid confusion when building designs. If we think a name could/should be changed, we should have that conversation with Design to keep synchronized. I'm 0/5 on this particular case.

Reply:
The point here is that the size is the same as small (icon size is identical), but the padding around the icon is less to allow the overall button to take up less space. Again, 0/5 on changing the name.

Reply:
ok, I am fine with that!